socorro/processor/mozilla_transform_rules.py

```diff
@@ -857,10 +858,14 @@ def __init__(self, config):
         try:
             sentry_dsn = self.config.sentry.dsn
         except KeyError:
-            # DotDict raises a KeyError when things are missing
+            # NOTE(willkg): DotDict raises a KeyError when things are missing
             sentry_dsn = None
+        self.sentry_dsn = sentry_dsn
+        self.generator = SignatureGenerator(error_handler=self._error_handler)
 
-        self.generator = SignatureGenerator(sentry_dsn=sentry_dsn)
+    def _error_handler(self, extra):
```

Comment:
I need to fix this--it needs to take the data it passed so it can add the crash id.